cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,574 @@ Cask Data Application Platform Release Notes
+  to include the CDAP classes before the Apache Twill classes.
+
+- :cask-issue:`CDAP-6384` - Updated the CDAP Standalone VM version of IntelliJ IDE to
+  2016.1.3.
+
+- :cask-issue:`CDAP-6386` - Fixed an issue where updating the properties of a dataset
+  deleted all of its partitions in Hive.
+
+- :cask-issue:`CDAP-6452` - Add a check for the environment variable
+  ``CDAP_UI_COMPRESSION_ENABLED`` to disable UI compression.
+
+- :cask-issue:`CDAP-6455` - Fixed the classpath of a MapReduce program launched by the
+  explore service to include the ``cdap-common.jar`` at the beginning.
+
+- :cask-issue:`CDAP-6486` - Fixed an issue that caused a Zookeeper watch to leak memory
+  every time a program was started.
+
+- :cask-issue:`CDAP-6510` - Fixed an issue where the ExploreService was attempting (with
+  no effect except for a slow down) to run the upgrade procedure for all explorable
+  datasets.
+
+- :cask-issue:`CDAP-6515` - Fixed classloading issues related to using Guava's
+  ``Optional`` class in Spark, allowing programs to perform left-outer and full-outer joins
+  on RDDs.
+
+- :cask-issue:`CDAP-6524` - Plugins now support the "char" primitive as a property type.
+
+- :cask-issue:`CDAP-6643` - Fixed an issue that caused massive log messages when there was
+  an underlying HDFS issues.
+
+- :cask-issue:`CDAP-6783` - Fixed the classpath ordering in Spark to load the classes from
+  ``cdap-common`` first.
+
+- :cask-issue:`CDAP-6829` - Fixed issues that prevented the Log Saver from performing
+  cleanup when metadata is present for a non-existing file.
+
+- :cask-issue:`CDAP-6852` - Fixed issues that makes the Log Saver more resilient to errors
+  while checkpointing.
+
+- :cask-issue:`CDAP-6860` - Improved performance in cube datasets when querying for more
+  than one measure in a query. This will also improve metrics query performance.
+
+- :cask-issue:`CDAP-6901` - Added a bootstrap step for authorization in CDAP. As part of
+  this step:
+
+    - 1. The user that CDAP runs as now receives "admin" privileges on the CDAP instance,
+    as well as "all" privileges on the system namespace.
+
+    - 2. The list of users specified in the parameter
+    ``security.authorization.admin.users`` in cdap-site.xml receives "admin" privileges on
+    the CDAP instance so that they can create namespaces.
+
+- :cask-issue:`CDAP-6929` - Logs from Spark driver and executors are now collected.
+
+- :cask-issue:`CDAP-6930` - **Incompatible Change:** Updated the "cdap-clients" to throw
+  ``UnauthorizedException`` when an operation returns ``403 - Forbidden`` from CDAP. Users
+  of "cdap-clients" may need to update their code to handle these exceptions.
+
+- :cask-issue:`CDAP-6935` - Fix a bug where the live-info endpoint was not working for
+  Workflows, MapReduce, Worker, and Spark.
+
+- :cask-issue:`CDAP-6939` - Fixed support in the CDAP UI for Google Chrome prior to
```

Comment:
Fixed support -- should this be -- Added CDAP UI support for Google Chrome releases prior to version 44.